### ISSUE TITLE
Toggle outline doesn't restore outline once hidden

### ISSUE THREAD

**Issue by msabramo** [frairon/go-outline#54]
If I do `Show/Hide Go Outline` (or the keyboard shortcut Ctrl+Alt+O) once, it shows the outline. If I do it a second time then the outline is hidden. If I do it a third time, it will not restore the outline. I have to quit and restart Atom to get the outline back.

**Atom Version**: 1.13.0
**Electron Version**: 1.3.13
**System**: Mac OS X 10.11.6
**go-outline Version**: 0.6.2

**Comment by msabramo** [comment]
If I have the Developer Tools open with the console showing, all I see is:

```
destroying panel           go-outline-view.coffee [sm]:346 destroying panel
```

when the outline is hidden but nothing else. No errors when I try to restore the outline.

**Comment by frairon** [comment]
yeah, that was bugging me for a long time too.
Should be fixed in v0.6.3, thanks a lot!
